### DISCUSSION TITLE
Any idea if this will work with the new "GD" sub model?

### DISCUSSION THREAD

**Discussion by GingerSnap-xx** [teeminus/NoTouchScreenFirmware#64] [Q&A] [answered]
They started shipping the screen with ST clones which is denoted with a GD in the model.
I received on of these clones and am trying to figure out if this firmware would work with the clones or not. I've been having trouble getting it to flash and I'm not sure if its the clone or human error e.g. my SD card being 8GB instead of "less than 8GB"

Gotta get muh klipper going

**Comment by teeminus** [comment]
I have no idea if it will work. I wouldn't be surprised if the clone devices are not updatable "in the field"...

**Reply by GingerSnap-xx** [reply]
Looking at the BTT firmware, the GD pre-built firmware sticks "GD" between "BIGTREE" and the model e.g. 
> BIGTREE_GD_TFT35_V3.0_E3.27.x

I'll adding that GD to the NoTouchScreenFirmware name and see if it flashes. After that I'll update with whether or not it bricks the screen 😆

**Reply by teeminus** [reply]
I think you can't brick the TFT when flashing from SD card. But that's just a guess 😅

**Reply by GingerSnap-xx** [reply]
You've spent enough time digging into the guts of this that I trust your guess.

**Reply by GingerSnap-xx** [reply]
After putting GD in the name it does try to load it, but it quickly blanks out the screen and freezes. Re-booting the printer does not revert back to old firmware. Re-flashing with the canned firmware from BTT now seems to be working fine.

**Reply by teeminus** [reply]
Maybe they changed other things as well on the new displays...

**Reply by GingerSnap-xx** [reply]
Maybe but I read some articles while struggling with similar woes on the MCU that claimed that the clones have quirks themselves that can break stuff built for the official STM chips.

**Comment by GingerSnap-xx** [comment, marked as answer]
As of 1.3.1 this does _not_ work with the GD variants of the TFT 35 E3
